api/client.go

```diff
@@ -351,6 +356,22 @@ func (c *Client) AddLocalCharm(curl *charm.URL, ch charm.Charm) (*charm.URL, err
 	return curl, nil
 }
 
+func validateCharmLXDProfile(ch charm.Charm) error {
```

Comment:
per conversation, suggesting moving to juju/juju/core/ , to be used in multiple places.